### ISSUE TITLE
Size for primary string attribute is not applied

### ISSUE THREAD

**Issue by koloboid** [balderdashy/sails#3134]
Hello. This simple case in model:

    module.exports = {
      attributes: {
        pk: {
          size: 22,
          type: 'string',
          primaryKey: true
        },
        name: {
          type: 'string',
          size: 50
        },
    }

gives a table in mysql with columns pk:varchar(255) and name:varchar(50), where as pk:varchar(22) and name:varchar(50) expected. I'm using latest sails 0.11.0
Thanks in advice

**Comment by sailsbot** [comment]
Thanks for posting, @koloboid.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
